### DISCUSSION TITLE
Are aliases supported by tabby?

### DISCUSSION THREAD

**Discussion by Krzysztof-Lempicki** [Eugeny/tabby#5707] [Q&A]
hi,

In cmder I can save in config my custom aliases:

`alias home='cd my/path'`

and run it in terminal.

How can achieve this with tabby?

**Comment by Eugeny** [comment]
Aliases are a shell feature - you can do the same in any terminal.

**Comment by Krzysztof-Lempicki** [comment]
I'm not talking about `alias` command.

![image](https://user-images.githubusercontent.com/55065779/153426259-07b688d2-f6fa-4d39-b073-fb0ccbb6a804.png)

This is how can it can be done in cmder.
Aliases are persisted and accessible in all terminals.

**Reply by Eugeny** [reply]
It looks like cmder is just [handing off this configuration to Doskey](https://github.com/cmderdev/cmder/wiki/Cmder-Aliases), but Tabby doesn't have UI for this.

**Reply by Krzysztof-Lempicki** [reply]
Will this be implemented?
It is very comfortable.

**Reply by xuanyanwow** [reply]
Is there any progress? We also hope to have this function
